src/Tests/Meditation.PatchingService.Tests/ProcessPatchingTests.cs

```diff
@@ -0,0 +1,75 @@
+using Meditation.Bootstrap.Managed;
+using Meditation.InjectorService;
+using Meditation.PatchingService.Models;
+using Meditation.TestUtils;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Meditation.PatchingService.Tests
+{
+    public class ProcessPatchingTests : TestsBase
+    {
+        private const string ipcTerminationSignaller = "/meditation/tests-ipc-signal";
+
+        [Theory]
+        [InlineData("net8.0")]
+#if WINDOWS
+        [InlineData("net472")]
+#endif
+        public async Task PatchingService_PatchOtherProcess(string netSdkIdentifier)
+        {
+            // Prepare
+            var patchingConfiguration = new PatchingConfiguration(
+                PatchInfo: new PatchInfo(
+                    Path: typeof(TestPatch.Harmony.Patch).Assembly.Location,
+                    TargetFullAssemblyName: typeof(TestPatch.Harmony.Patch).Assembly.FullName ?? "test-assembly",
+                    Method: null! /* Not used by patch applier */),
```

Comment:
I think it would be more comfortable to read if PatchInfo will be saved into the variable, and then used in PatchingConfiguration